test/models/compute_resources/ovirt_test.rb

```diff
@@ -13,6 +13,18 @@ class OvirtTest < ActiveSupport::TestCase
     assert_equal host, as_admin { cr.associated_host(vm) }
   end
 
+  describe 'validate' do
+    test 'valid if url is present' do
```

Comment:
`:url_schema => ['http', 'https']` should be only applicable for ovirt? Or should be applied for rackspace, openstack as well?